app/src/app/env.js

```diff
@@ -1,9 +1,13 @@
 import {env} from '../../package.json'
 import rc from './rc'
 
-if (rc.domain) {
+if (Object.prototype.hasOwnProperty.call(rc, 'domain')) {
   env.host.domain = rc.domain
   env.chooseDomainDisabled = true
 }
 
+if (Object.prototype.hasOwnProperty.call(rc, 'startInBackgroundWhenAutostarted')) {
```

Comment:
same here

Reply:
`startInBackgroundWhenAutostarted` is default `true` and when the users sets it to `false` in `graperc`, the if block would still not be evaluated if i just check `if(rc.startInBackgroundWhenAutostarted)`

Reply:
I think you didn't understand what is going on there. Please reevaluate, you don't need this.

Reply:
What can I use instead of `hasOwnProperty` to assert that an object has a certain property? The `in` keyword in JS is not what you would expect and [causes problems](http://adripofjavascript.com/blog/drips/the-uses-of-in-vs-hasownproperty.html).

just checking `rc.startInBackgroundWhenAutostarted` doesn't assert the availability of the property but evaluates the value of the property.

Reply:
I could do 

```js
env.startInBackgroundWhenAutostarted = rc.startInBackgroundWhenAutostarted || env.startInBackgroundWhenAutostarted;
```

or even 

```
env.startInBackgroundWhenAutostarted = rc.startInBackgroundWhenAutostarted || true;
```

(and remove from the env files)

would you prefer this?

Reply:
If we need that flag at all. Again, do we have a use case where it autostarted but should go full mode?

Reply:
Felix said it would be nice to have